### PR TITLE
space

### PR DESCRIPTION
**Pull request by aramlaka**

### CODE REVIEW

**Comment by Chasesc** [comment]
This is a breaking change. `config.get` **MUST** line up with `IMG_SAVE_PATH` otherwise everything will break & all order is lost. This would be disastrous.

The newline at the end is good though.

File: bot.py [hunk 1]
```diff
@@ -16,7 +16,7 @@
 START_TIME = datetime.now()
 
 IMG_SAVE_PATH       = Path(config.get('save_path'))
-SUPPORTED_FILETYPES = set(config.get('supported_filetypes'))
+SUPPORTED_FILETYPES = set( config.get('supported_filetypes'))
```
Comment (Chasesc) [inline bot.py:19]: This is evil

Comment (Chasesc) [reply bot.py:19]: haha the matrix xd

**Comment by PhillipHunter** [comment]
What business value does this provide to the project?

**Pull request closed without merging**